pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -298,7 +299,9 @@ func (sf *streamIngestionFrontier) Next() (
 			// If heartbeatSender has error, it means remote has error, we want to
 			// stop the processor.
 		case <-sf.heartbeatSender.stoppedChan:
-			sf.MoveToDraining(sf.heartbeatSender.err())
+			err := sf.heartbeatSender.err()
+			log.Warningf(sf.Ctx, "heartbeat sender has stopped with error: %s\n", err)
```

Comment:
I don't think we typically want to attach newlines to our log messages. The logging system will do that for us..